Extensions/Emails/Services/EmailService.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.Linq;
+using Lombiq.HelpfulExtensions.Extensions.Emails.Models;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using OrchardCore.Email;
+using OrchardCore.Environment.Shell.Scope;
+
+namespace Lombiq.HelpfulExtensions.Extensions.Emails.Services
+{
+    public class EmailService : IEmailService
+    {
+        public void SendEmailDeferred(EmailParameters parameters) =>
+            ShellScope.AddDeferredTask(async scope =>
+            {
+                var smtpService = scope.ServiceProvider.GetService<ISmtpService>();
+                var logger = scope.ServiceProvider.GetService<ILogger<EmailService>>();
+                var result = await smtpService.SendAsync(new MailMessage
+                {
+                    Sender = parameters.Sender,
+                    To = parameters.To?.Join(","),
+                    Cc = parameters.Cc?.Join(","),
+                    Bcc = parameters.Bcc?.Join(","),
+                    Subject = parameters.Subject,
+                    ReplyTo = parameters.ReplyTo,
+                    Body = parameters.Body,
+                    IsBodyHtml = true,

```

Comment:
It should be mentioned in either the docs or the method name that it's for HTML email because it's not obvious nor the default.